logicrunner/builtin/contract/member/member.go

```diff
@@ -219,20 +223,20 @@ func getBalanceCall(params map[string]interface{}) (interface{}, error) {
 }
 func (m *Member) transferCall(params map[string]interface{}) (interface{}, error) {
 
-	toMemberReferenceI, ok := params["toMemberReference"].(string)
+	recipientReferenceStr, ok := params["toMemberReference"].(string)
 	if !ok {
-		return nil, fmt.Errorf("incorect input: failed to get 'toMemberReference' param")
+		return nil, fmt.Errorf("incorect input: failed to get 'recipientReference' param")
```

Comment:
You try to get `toMemberReference ` param, but error message says `recipientReference `